test/mdx-test.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+#
+# Run an integration test suite using mdx
+# See the tla/cli-integration-tests.md for details.
+#
+# Shon Feder, 2020
```

Comment:
Could you comment on why you need such a long script to run the tests? Does it mean that we probably should change the format of our tests?

Reply:
The script maybe overly engineered or overly defensive.

This script does a few things:

1. Sequences two command executions
2. Ensures pretty feedback on test failure
3. Provides some defensive automation to protect against misuse or common errors
4. Manages paths in a general way, to prevent polluting our working directory with test artifacts

If don't care about those 2-3, then we could just hardcode our target into the Makefile with two command invocations.

Are there any parts of the wrapper script find unnecessary, hard to understand, or otherwise objectionable?

Reply:
No objections. Just wondered about the code size :)

Reply:
It's a good question. I should probably put this comment in the code, for when I look back in a month and wonder "why is this so complicated"? :)